singleFrameProcCoreTasks/charimg.rst

```diff
@@ -0,0 +1,304 @@
+#####################
+CharacterizeImageTask
+#####################
+
+Given an exposure with defects repaired (masked and interpolated over,
+e.g. as output by :doc:`IsrTask <isrtask>`), this task does initial
+source extraction and PSF estimation.
+
+
+Its primary functions are to:
+
+  - Detect and measure bright sources
+
+  - Repair cosmic rays
+
+  - Measure and subtract background
+
+  - Measure the PSF
```

Comment:
Yes, i know that routine in the code, and i did have that description in at some point.  I'll go back and find that, but i still don't know that it needs to go at the very top, where we're trying to be brief..?